package.json

```diff
@@ -1,5 +1,12 @@
 {
-    "dependencies": {
-        "yuglify": "^0.1.4"
-    }
+  "scripts": {
+    "test": "eslint dashboard/static/js/*.js; csslint --quiet --warnings=' ' dashboard/static/css/*.css push/static/css/*.css"
```

Comment:
Yep, that would be similar to what we did in the Gulp script:

https://github.com/openjck/push-dev-dashboard/blob/jshint/gulpfile.js#L9

I'll make the update